content/en/continuous_integration/tests/javascript.md

```diff
@@ -323,6 +350,28 @@ it('renders a hello world', () => {
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][105] section of the Node.js custom instrumentation documentation.
 
+### Adding custom metrics to tests
+
+To add custom metrics to your tests, such as the memory allocations, use `cy.task('dd:addTags', { yourTags: 'here' })` in your test or hooks.
+
+For example:
+
+```javascript
+beforeEach(() => {
+  cy.task('dd:addTags', {
+    'before.each':
+    'certain.information'
```

Comment:
I know this is like this in the custom tags section, but let's use the chance to change it to this: 
```suggestion
    'before.each': 'certain.information'
```